fpx-cli/Cargo.toml

```diff
@@ -0,0 +1,76 @@
+[package]
+name = "fpx-cli"
+version = { workspace = true }
+edition = "2021"
+authors = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+
+[features]
+embed-studio = [] # When enabled it will embed Studio from frontend/dist
```

Comment:
Currently this is not working anymore, so we might just get rid of it. Because this features requires that studio would have been build (ie. the `dist` folder needed to exists) I wanted to make it a optin feature (so that everyone can easily use the fpx-cli). 

There are two situations where you'd use a Rust feature: either in your `Cargo.toml`, where along a dependency you'd ask for a feature. But in this case you'd use something like: `cargo build -F embed-studio` to build the cli with the studio embedded in there. 

For another example, there is the fpx-app, which allows for the TS or Rust API to be enable using a flag.